models/schema.yml

```diff
@@ -279,3 +279,101 @@ models:
         description: "The number of units in inventory of the product variant"
         tests:
           - not_null
+
+metrics:
+  - name: number_customers
+    label: Number of Customers
+    model: ref('shopify__customers')
+    description: "The number of customers"
+
+    type: count_distinct
+    sql: customer_id
+
+    timestamp: created_at_timestamp
+    time_grains: [ day, week, month, year ]
+
+    dimensions:
+      - shipping_address_city
+      - shipping_address_country
+
+  - name: number_orders
+    label: Number of Orders
+    model: ref('shopify__orders')
+    description: "The number of orders"
+
+    type: count_distinct
+    sql: order_id
+
+    timestamp: created_at_timestamp
+    time_grains: [ day, week, month, year ]
+
+    dimensions:
+      - customer_id
+      - shipping_address_city
+      - shipping_address_country
+
+  - name: number_items_sold
+    label: Number of Items Sold
+    model: ref('shopify__order_line_items')
+    description: "The number of items sold"
+
+    type: count_distinct
+    sql: order_line_item_id
+
+    timestamp: created_at_timestamp
+    time_grains: [ day, week, month, year ]
+
+    dimensions:
+      - order_id
+      - shipping_address_city
+      - shipping_address_country
+
+  - name: average_order_value
+    label: Average Order Value
+    model: ref('shopify__orders')
+    description: "The average order value"
+
+    type: average
+    sql: total_price
+
+    timestamp: created_at_timestamp
+    time_grains: [ day, week, month, year ]
+
+    dimensions:
+      - customer_id
+      - shipping_address_city
+      - shipping_address_country
+
+  - name: average_order_quantity
+    label: Average Order Quantity
+    model: ref('shopify__order_line_items')
```

Comment:
Do it per the orders table

Reply:
this I did via shopify__order_line_items so that we could see the quantity of items sold in the same table but I guess thats not going to be needed